### ISSUE TITLE
Please consider a GraphQL interface.

### ISSUE THREAD

**Issue by austinsmorris** [varsitynewsnetwork/wordpress-rest-api-client#10]
k thx bai

**Comment by mrkrstphr** [comment]
can we get some SOAP in here too? i need more wsdl in my life.

**Comment by austinsmorris** [comment]
@boboudreau , plz implement asap.  k thx bai.

**Comment by mrkrstphr** [comment]
Should probably rename this organization, y'all.

**Comment by ryanplasma** [comment]
plz

**Comment by ryanplasma** [comment]
+1

**Comment by fontebasso** [comment]
@boboudreau For this to be possible I believe that who should implement the feature be the same WordPress I am right? Since the interface we have with the data is via REST is not it?

**Comment by mrkrstphr** [comment]
Yeah, yeah. I think @fontebasso has the right idea here. Thoughts @boboudreau?
